marker-vert.glsl

```diff
@@ -38,7 +38,7 @@ void main() {
   float size = size * maxSize / 255.;
   float borderSize = borderSize * maxSize / 255.;
 
-  gl_PointSize = 2. * size * pixelRatio;
+  gl_PointSize = 2. * size * 2.;
```

Comment:
Why not fixing pixel ratio from uniforms side?

Reply:
Thanks @dy for looking at this.
It seems to be used in one other place in the shader: https://github.com/gl-vis/regl-scatter2d/blob/6a8f3576467f41bc2f8d6a4e78c646e933532439/marker-vert.glsl#L42.

Reply:
But fragPointSize is different from gl_PointSize. Can you possibly add a test-case for the issue?

Reply:
> Why not fixing pixel ratio from uniforms side?

`pixelRatio` could be set as a constant of 2 here: https://github.com/gl-vis/regl-scatter2d/blob/091a1a70296dbb81cea4a320812e62ba6e928353/scatter.js#L84; but I thought it may be more consistent e.g. with other gl-vis/3d modules where other `pixelRatio` values are possible.

Reply:
That should be done as a separate option if you want to make some default value. Here https://github.com/gl-vis/regl-scatter2d/blob/master/scatter.js#L199

Reply:
@dy Our original issue is presented in [this codepen](https://codepen.io/MojtabaSamimi/pen/OqwJBM?editors=0010).
I tried the other option that you mentioned; but what is really needed for us to be able to pass various `pixelRatio` values (not a constant) and get quite similar marker size.

Reply:
Not sure I understand. Why not avoiding passing `plotGlPixelRatio` as `pixelRatio` at all?
The cases mentioned in the codepen make sure the point size is consisted in devices with various `pixelRatio`. If you want to pass "fake" pixelRatio - the point size will be increased, that's what expected. Am I getting it wrong?

Reply:
In Plotly.js `plotGlPixelRatio` is the same as `pixelRatio`. Here is [another codepen example](https://codepen.io/MojtabaSamimi/pen/ZZOZjV?editors=1000) comparing `scatter`, `scattergl` & `scatter3d` with different pixelRatio values lines and markers. The only module which treat pixelRatio differently seems to be this module. When passing different pixelRatio values we don't expect a change in point sizes but slightly higher render quality.

Reply:
Have you considered that on the devices with another default pixelRatio? The point size is likely smaller there, other gl- components don’t consider that.

Reply:
The codepens on different devices appear to be rendered OK with this fix.
@etpinard what's your thoughts here.
Should we consider another fix something like this? https://github.com/plotly/plotly.js/compare/markersize-second-approach

Reply:
Thanks for your hard work @archmoj - your patch appear to make `scattergl` consistent with how markers are drawn in `scatter3d` so :ok_hand: 

As this repo remains in @dy 's hands, we could perhaps make this new/consistent `pixelRatio` behaviour an opt-in e.g. by adding a boolean flag to `regl-scatter2d` that we would turn on when instantiating in plotly.js